sleap/gui/commands.py

```diff
@@ -1601,6 +1605,28 @@ class GoPrevSuggestedFrame(GoNextSuggestedFrame):
     seek_direction = -1
 
 
+class GoNextInstanceChange(NavCommand):
+    @classmethod
+    def do_action(cls, context: CommandContext, params: dict):
+        video = context.state["video"]
+        cur_idx = context.state["frame_idx"]
+
+        # Get current number of instances
+        current_frame = context.labels.find(video, cur_idx, return_new=True)[0]
+        current_instance_count = len(current_frame.instances)
```

Comment:
`LabeledFrame.instances` returns both the user instances and underlying predictions (i.e. when a user double clicks a prediction to correct it, the prediction will still be saved in the `instances` list, but the user only sees `instance_to_show`:
https://github.com/talmolab/sleap/blob/653144760882aa3f5c7f1199366d45aba0bccaed/sleap/instance.py#L1677-L1698